README.md

```diff
@@ -30,10 +30,11 @@ A low-level library to play sound.
 
 ## Prerequisite
 
-On most platforms you will need a C/C++ compiler in your path that Go can use.
+On some platforms you will need a C/C++ compiler in your path that Go can use.
 
-- Windows: [MingW](https://www.mingw-w64.org/downloads/#mingw-builds) or similar
-- Mac/Linux: Should be installed by default, but if not try [GCC](https://gcc.gnu.org/) or [Clang](https://releases.llvm.org/download.html)
+- Linux: Should be installed by default, but if not try [GCC](https://gcc.gnu.org/) or [Clang](https://releases.llvm.org/download.html)
```

Comment:
Linux and other UNIX systems (as macOS is also one of UNIX, let's put macOS first and then put this item second)

Reply:
OK renamed Linux and put MacOS bullet point first